ui/plugins/failure_summary/main.html

```diff
@@ -63,7 +63,23 @@
 
         </li>
 
-        <li ng-if="!tabs.failureSummary.is_loading && jobLogsAllParsed && bugSuggestionsLoaded && job_log_urls.length && suggestions.length == 0">
+        <span ng-if="errors.length > 0">
```

Comment:
Most of the responses in this panel have a blue background with a certain size text.  This text is a bit too large.  Would you please reuse the CSS classes that are used elsewhere to make the text visually uniform?

I think after these changes, we can go ahead and merge this.